CONTRIBUTING.md

```diff
@@ -17,10 +16,79 @@ There are several ways you can contribute :
 
 - **You want to help, but don't know what to do?** Check the [TODO list](https://github.com/neveldo/jQuery-Mapael/wiki/TODO-list) for features/issues we need/are working on.
 
-_ _ _
+Here are some readings: <https://guides.github.com/activities/contributing-to-open-source>
 
-Here are some readings:
+---
 
-1. <https://guides.github.com/activities/contributing-to-open-source>
+# Dev information
 
-2. <http://kbroman.org/github_tutorial/pages/fork.html>
+## Cloning the repository
+
+Before making any modification to Mapael, you will need your own Mapael fork!
+
+Here are some reading to get a local Mapael:
+1. <https://help.github.com/articles/fork-a-repo/>
+1. <http://kbroman.org/github_tutorial/pages/fork.html>
+
+## Environment
+
+Mapael uses [NPM](https://www.npmjs.com/) to manage package dependencies 
+and [Grunt](https://gruntjs.com/) to run development tasks.
+
+After cloning locally the Mapael repository, run this command-line:
+```
+npm install
+```
+
+Then you will be able to run all related Grunt tasks.
+
+## Code quality
+
+Mapael uses [JSHint](http://jshint.com/about/) to check code quality.
+The rules are [explained in the Wiki](https://github.com/neveldo/jQuery-Mapael/wiki/JSHint).
+
+You can run this check with the following command-line:
+```
+grunt jshint
+```
+
+## Test suite
+
+There is a test suite in the `/test` folder, power by [QUnit](http://qunitjs.com/).
+While not broken, it is however, it is very light. We sadly don't have the time to expand it for now.
```

Comment:
Maybe a small mistake here ?

it is however, it is very light. => it is however very light.

Reply:
Fixed!